manifests/b/buchen/portfolio/0.69.1/buchen.portfolio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: buchen.portfolio
+PackageVersion: 0.69.1
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/portfolio-performance/portfolio/releases/download/0.69.1/PortfolioPerformance-0.69.1-setup.exe
+  InstallerSha256: 90663C6F206AA33B2146C6A4B49ACE368133E450C82FABBEEFD2C769549F2A8F
```

```suggestion
  InstallerSha256: 90663C6F206AA33B2146C6A4B49ACE368133E450C82FABBEEFD2C769549F2A8F
  ProductCode: Portfolio Performance
```

Metadata for better package correlation